moped-database/seeds/1602292389297_initial_seed_staging.sql

```diff
@@ -998,204 +998,6 @@ INSERT INTO public.moped_project_files (project_file_id, project_id, file_key, f
---
-
-SELECT pg_catalog.setval('public.moped_proj_fund_source_proj_fund_source_id_seq', 1, false);
-
-
-
---
--- TOC entry 6132 (class 0 OID 0)
--- Dependencies: 325
--- Name: moped_proj_milestones_project_milestone_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_proj_milestones_project_milestone_id_seq', 1, false);
-
-
---
--- TOC entry 6133 (class 0 OID 0)
--- Dependencies: 297
--- Name: moped_proj_notes_project_note_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_proj_notes_project_note_id_seq', 2, true);
-
-
---
--- TOC entry 6134 (class 0 OID 0)
--- Dependencies: 299
--- Name: moped_proj_partners_proj_partner_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_proj_partners_proj_partner_id_seq', 1, false);
-
-
---
--- TOC entry 6135 (class 0 OID 0)
--- Dependencies: 300
--- Name: moped_proj_personnel_project_personnel_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_proj_personnel_project_personnel_id_seq', 212, true);
-
-
-
---
--- TOC entry 6138 (class 0 OID 0)
--- Dependencies: 317
--- Name: moped_project_files_project_file_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_project_files_project_file_id_seq', 32, true);
-
-
---
--- TOC entry 6139 (class 0 OID 0)
--- Dependencies: 307
--- Name: moped_project_project_id_simple_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_project_project_id_simple_seq', 226, true);
-
-
---
--- TOC entry 6140 (class 0 OID 0)
--- Dependencies: 309
--- Name: moped_project_roles_project_role_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_project_roles_project_role_id_seq', 13, true);
-
-
-
-
---
--- TOC entry 6143 (class 0 OID 0)
--- Dependencies: 313
--- Name: moped_status_status_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_status_status_id_seq', 4, true);
-
-
---
--- TOC entry 6144 (class 0 OID 0)
--- Dependencies: 319
--- Name: moped_subphases_subphase_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_subphases_subphase_id_seq', 1, false);
-
-
---
--- TOC entry 6145 (class 0 OID 0)
--- Dependencies: 257
--- Name: moped_users_user_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_users_user_id_seq', 45, true);
-
-
---
--- TOC entry 6146 (class 0 OID 0)
--- Dependencies: 315
--- Name: moped_workgroup_workgroup_id_seq; Type: SEQUENCE SET; Schema: public; Owner: atd_moped
---
-
-SELECT pg_catalog.setval('public.moped_workgroup_workgroup_id_seq', 1, true);
-
-
```

Comment:
@frankhereford - do i understand correctly: here is where you nix the manual setting of the sequence pointer so as to let the DB handle this automatically?

this only affects our local and heroku builds—since these migrations have already been applied to stating and prod. the pointers in staging and prod are being tweaked in your new migration within this PR